addons/project/static/src/views/project_task_list/project_task_list_controller.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module */
+
+import { ListController } from "@web/views/list/list_controller";
+import { DeleteSubtasksConfirmationDialog } from "@project/components/delete_subtasks_confirmation_dialog/delete_subtasks_confirmation_dialog";
+import { sprintf } from "@web/core/utils/strings";
+import { session } from "@web/session";
+
+export class ProjectTaskListController extends ListController {
+    async onDeleteSelectedRecords() {
+        if (!Math.max(...this.model.root.selection.map((record) => record.data.subtask_count ))) {
+            return this.deleteSelectedRecords();
+        }
+        this.dialogService.add(DeleteSubtasksConfirmationDialog, {
+            confirm: this.deleteSelectedRecords.bind(this)
+        });
+    }
+
+    async deleteSelectedRecords() {
+        const root = this.model.root;
+        const total = root.count;
+        const resIds = await root.deleteRecords();
+        this.model.notify();
```

Comment:
I see a PR to remove the `notify` method, so maybe it is no longer needed since we have now the new relational model. 🙂

Reply:
> I see a PR to remove the `notify` method, so maybe it is no longer needed since we have now the new relational model. slightly_smiling_face

Indeed, I've removed that part :+1: